experimental/extractors.md

```diff
@@ -0,0 +1,29 @@
+# [Extractors for ECMAScript][proposal-extractors]
+
+## QualifiedName
+
+```js
+interface QualifiedName <: Node {
+    type: "QualifiedName";
+    left: QualifiedName | Identifier;
+    right: Identifier;
+}
+```
+An Identifier (`foo`) or dotted-name (`foo.bar`).
```

Comment:
Does this mean `left` can also be undefined?

Reply:
No, `left` must be either an `Identifier` (`foo`) or a qualified name (`foo.bar`). The current AST design is borrowed from Babel's `TSQualifiedName`.

Reply:
Oh sorry, this note confused me due to its location. I see below in ObjectPattern and ArrayPattern that we’ve just added QualifiedName in addition to Identifier. 

Can we remove this paragraph? It doesn’t seem to clarify what’s already otherwise on this page.